.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{

```

Comment:
I'm not familiar with what we're having in the repo for VSCode settings, was this supposed to be committed? If so what does it do?

Reply:
I'd like to add the launch settings so it's possible to use the integrated debugger during testing - so you can add breakpoints etc. Just open the debugger pane (striked-through bug icon, see screenshot below) select the configuration & click play. (No config in the screenshot)
![screenshot_debugger_pane](https://user-images.githubusercontent.com/3046542/50685141-57cfb380-1018-11e9-85b0-d5143f7febf0.PNG)

OK, I'll check the other test files and also disable flowjs there.

Reply:
OK, I've fixed all linting errors. Also added two unrelated flow linting errors.